homeassistant/components/efesto/climate.py

```diff
@@ -0,0 +1,290 @@
+"""Support for Efesto heating devices."""
+import logging
+
+from efestoclient import (  # pylint: disable=redefined-builtin
+    ConnectionError,
+    EfestoClient,
+    Error as EfestoError,
+    InvalidURLError,
+    UnauthorizedError,
+)
+
+from homeassistant.components.climate import ClimateDevice
+from homeassistant.components.climate.const import (
+    CURRENT_HVAC_HEAT,
+    CURRENT_HVAC_IDLE,
+    CURRENT_HVAC_OFF,
+    HVAC_MODE_HEAT,
+    HVAC_MODE_OFF,
+    SUPPORT_FAN_MODE,
+    SUPPORT_TARGET_TEMPERATURE,
+)
+from homeassistant.const import (
+    ATTR_TEMPERATURE,
+    CONF_DEVICE,
+    CONF_PASSWORD,
+    CONF_URL,
+    CONF_USERNAME,
+    PRECISION_WHOLE,
+    TEMP_CELSIUS,
+)
+
+from .const import (
+    ATTR_DEVICE_STATUS,
+    ATTR_HUMAN_DEVICE_STATUS,
+    ATTR_REAL_POWER,
+    ATTR_SMOKE_TEMP,
+    DOMAIN,
+    FAN_1,
+    FAN_2,
+    FAN_3,
+    FAN_4,
+    FAN_5,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+FAN_MODES = [
+    FAN_1,
+    FAN_2,
+    FAN_3,
+    FAN_4,
+    FAN_5,
+]
+
+CURRENT_HVAC_MAP_EFESTO_HEAT = {
+    "ON": CURRENT_HVAC_HEAT,
+    "CLEANING FIRE-POT": CURRENT_HVAC_HEAT,
+    "FLAME LIGHT": CURRENT_HVAC_HEAT,
+    "OFF": CURRENT_HVAC_OFF,
```

Comment:
Please create constants for the library specific modes in const.py and import them here.